src/main/resources/application.properties

```diff
@@ -1,6 +1,5 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store
-spring.datasource.username=root
-spring.datasource.password=test
```

Comment:
is it possible to run your application without docker? Better return these properties